### PR TITLE
feat: added new storyblok rich text API

### PR DESCRIPTION
**Pull request by alvarosabu**
This PR is to integrate `@storyblok/richtext` into the vue SDK via `@storyblok/js`. 

- [x] `useStoryblokRichText` composable
- [x] Dedicated `StoryblokRichText.vue` component
- [x] Types
- [x] Docs
- [x] Component test

Closes PRO-309

### CODE REVIEW

File: playground/pages/Home.vue [hunk 1]
```diff
@@ -11,12 +11,69 @@ const { data } = await storyblokApi.get("cdn/stories/vue", {
 
 const story = ref(null);
 story.value = data.story;
-
+console.log(story.value);
```
Comment (alexjoverm) [inline playground/pages/Home.vue:14]: necessary to keep?

File: playground/pages/Home.vue [hunk 2]
```diff
@@ -11,12 +11,69 @@ const { data } = await storyblokApi.get("cdn/stories/vue", {
...
 onMounted(() => {
   useStoryblokBridge(story.value.id, (evStory) => (story.value = evStory));
 });
+
```
Comment (alexjoverm) [inline playground/pages/Home.vue:18]: Are we planning to use this commented code later in time?

**Review by alexjoverm — APPROVED** [review]

File: playground/pages/Home.vue (hunk 1 above)
Comment (alvarosabu) [reply playground/pages/Home.vue:14]: This code is located on our playground for debugging purposes

File: playground/pages/Home.vue (hunk 2 above)
Comment (alvarosabu) [reply playground/pages/Home.vue:18]: Is there for debugging purposes.

**Pull request merged**